problems/countThrees.js

```diff
@@ -13,8 +13,9 @@
 * returns: 0
 */
 
-function countThrees() {
-
+function countThrees(n) {
+     n.forEach
```

Comment:
Nice try. forEach is an array method. Here we only have n which is number. Try converting n into a string and looping through with a variable that tracks how often a character is "3"